events/c_2_project_and_community.toml

```diff
@@ -64,81 +66,39 @@ This is a track
 # - recommended lunch: 12:00 or 13:00
 # - recommended end:   17:00
 # - use 15m, 20m, 30m, 45m, or 60m session slots
-# - use 15m breaks, one in the morning, one or two in the afternoon
+
 [[timeslots]]
 startTime="09:00"
-speakers=["@speaker_name", "@other_speaker_name"]
+speakers=["@mishmosh"]
 title="Welcome standup discussion"
-description=""
+description="A brief introduction to the track"
 
 [[timeslots]]
-startTime="09:30"
-speakers=["@yourname"]
-title="Track Intro"
-description=""
+startTime="09:15"
+speakers=["@lidel"]
+title="Lightweight RFC Process"
+description="A walkthrough of the new RFC process and its motivations, updated specs repository, and how you can submit an RFC."
```

Comment:
pumped for this one. I still need to read https://github.com/ipfs/specs/pull/289

[I have a history with setting up an RFC process](https://github.com/qri-io/rfcs/), and would recommend covering at least a few things in this talk:

1. Guidance for community members on _when_ to write an RFC, in contrast to when to file a pull request on a specific implementation.
2. Try to sell folks on why they should write an RFC, being explicit about benefits. (maybe ipfs.io _has_ to follow the spec, and this is the way to finally deal with getting stuff into that specific gateway? cc @aschmahmann)
3. Try to write lay out a specific plan of how we're going to get the community to engage with this.